saq/queue.py

```diff
@@ -729,6 +729,7 @@ def __init__(self, pubsub: PubSub, prefix: str):
         self._subscriptions: t.Dict[bytes, t.Set[Q]] = defaultdict(set)
         self._queues: t.Dict[Q, t.Set[bytes]] = {}
         self._daemon_task: t.Optional[asyncio.Task] = None
+        self._daemon_creation_lock = asyncio.Lock()
```

Comment:
Stylistically, we aim for concise names.

How about `self._creation_lock`, or even just `self._lock`, since this is the only one?

Reply:
fix